.github/workflows/ci-build.yaml

```diff
@@ -39,10 +39,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [ windows-2019, windows-2022 ]
-        php: [ "8.2", "8.1", "8.0", "7.4", "7.3", "7.2", "7.1" ]
+        php: [ "8.3", "8.2", "8.1", "8.0", "7.4", "7.3", "7.2", "7.1" ]
```

Comment:
Needs upstream PRs cmb69/setup-php-sdk#22 or cmb69/setup-php-sdk#23 to have support